### ISSUE TITLE
When in parent of section show the open menu

### ISSUE THREAD

**Issue by carasmo** [aptuitiv/ap-drilldown-menu#1]
Is there a way that when the user has navigated to the page, for example, "about" that the children of about is shown rather than the main top level of the menu?

**Comment by erictompkins** [comment]
Christina, 

I understand what you’re asking for and we’ve thought about the same thing. However, since this library consolidates a multi-link desktop navigation menu to only be accessible under a single link on a mobile device I would be concerned about usability by doing this and having the user be confused about how to get to different areas of the website. 

Have you seen this done in other libraries?

**Comment by carasmo** [comment]
Yes, in a way, if you go to this page on a narrow device or narrow your viewport:

http://mmenu.frebsite.nl/tutorials/advanced/styling-lists.html

This menu shows the section you're in and the title of that section.

**Comment by erictompkins** [comment]
We will look into making this an option. I'm not sure that I want it to be the default, but an option is fine. 
Thank you for using our plugin.
